app/models/spree/gift_card.rb

```diff
@@ -214,7 +217,7 @@ def amount_remaining_is_positive
     end
 
     def able_to_redeem?(user)
-      Spree::Config.allow_gift_card_redeem && user && user.email == email && amount_remaining.to_f > 0.0 && line_item.order.completed?
+      Spree::Config.allow_gift_card_redeem && user && user.email == email && amount_remaining.to_f > 0.0 && (line_item&.order&.completed? || self.created_by.eql?('admin'))
```

Comment:
why we make this change?

Reply:
I did this to distinguish between gift cards created by users and those created by admins,